sub-packages/bionemo-noodles/src/bionemo/noodles/nvfaidx.py

```diff
@@ -33,7 +33,15 @@ class SequenceAccessor:
     """
 
     def __init__(self, reader: PyIndexedMmapFastaReader, seqid: str, length: int) -> None:
-        """Construct a SequenceAccessor object.
+        """Construct a SequenceAccessor object. Ultimately this is used as a convenience object with NvFaidx.
+
+        When querying the following are true:
+            - Negative indexing is supported, but it does not wrap. so query[-10000] for a sequence of length 1 will fail.
+            - out of bounds indexing is truncated: query[1:999999999] will return a string from position 1 to the terminus.
```

Comment:
What is the behavior if the start of the interval is also beyond the end of the string? Is empty string returned or is this a failure?

Reply:
added this to the docstring and a supporting test. (it returns the empty string).